src/instrumenter/transpiling_context.ts

```diff
@@ -35,11 +35,9 @@ import { makeTypeString } from "./type_string";
 import { FactoryMap, ScribbleFactory, StructMap } from "./utils";
 
 export enum InstrumentationSiteType {
-    FunctionAnnotation,
-    ContractInvariant,
-    UserDefinedFunction,
-    StateVarUpdated,
-    Assert
+    TwoPointWrapper,
+    SinglePointWrapper,
+    Custom
```

Comment:
I'm wondering if it is possible to document these values to state their meaning and purpose. WDYT?

Reply:
Fixed in [93c526e](https://github.com/ConsenSys/scribble/pull/154/commits/93c526e31480369c29e5ddbdbf8965f5ea7bfd73)